### PR TITLE
Plan migration fix

### PR DESCRIPTION
**Pull request by AndriiMysko**
Hello @artursmirnov !
A hotfix for migrating from a legacy plan to a new one.
I've also tried to fix the problem mentioned in the SCA fix, which turned out to be early task cancellation issue and is now working properly. Please take a look.

### CODE REVIEW

File: app/components/billing/account.js [hunk 1]
```diff
@@ -31,6 +34,12 @@ export default Component.extend({
 
   showPlansSelector: false,
   showAddonsSelector: false,
+  isTrialProcessCompleted: computed(function () {
+    return !this.isTrial;
+  }),
+  isEduProcessCompleted: computed(function () {
+    return !this.isEducation;
+  }),
```
Comment (artursmirnov) [inline app/components/billing/account.js:42]: Shouldn't these properties track changes on dependent ones? Or are they just initialization values?
```suggestion
  isTrialProcessCompleted: not('isTrial'),
  isEduProcessCompleted: not('isEducation'),
```

**Review by artursmirnov — APPROVED** [review]
Hey @AndriiMysko thank you for the fix and for revisiting the async behavior issue! It looks very good to me, I have just one little concern below. If that's intended, please let me know and I'll merge the PR 👍

File: app/components/billing/account.js (hunk 1 above)
Comment (AndriiMysko) [reply app/components/billing/account.js:42]: They are just initialisation values, that's correct!

Comment (artursmirnov) [reply app/components/billing/account.js:42]: Thanks, I thought it would be the case 👍

**Pull request merged**